docs/usage.md

```diff
@@ -457,6 +457,27 @@ spec:
 ...
 ```
 
+>**Note:** If you want to specify the ObjectBucket name in configMap or secret, you can use envFrom and specify environments variable in `ObjectBucketName` like below.
```

Comment:
Do you mean the below?
```suggestion
>**Note:** If you want to specify the ObjectBucket name in a ConfigMap or Secret, you can use `envFrom` and specify the environment variable name in `jobConfig.bucketConfig.bucketName` as follows.
```

Reply:
@ymmt2005 
Yes, thank you for pointing that out.